fastlane_bot/config/network.py

```diff
@@ -723,7 +724,7 @@ class _ConfigNetworkMantle(ConfigNetwork):
     # WRAPPED_GAS_TOKEN_KEY = "WETH-0006"
     # STABLECOIN_KEY = "USDC-2913"
 
-    NATIVE_GAS_TOKEN_ADDRESS = "0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000"
+    NATIVE_GAS_TOKEN_ADDRESS = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
```

Comment:
I'm still seeing this in `fastlane_bot/data/blockchain_data/mantle/tokens.csv`:
```
0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000,18,MNT
```

Reply:
These were updated.